caffe2/operators/segment_reduction_op.cc

```diff
@@ -64,150 +64,254 @@ string FormatDoc() {
   return doc;
 }
 
+// Helper function to enforce naming conventions at compile time.
```

Comment:
Would you prefer that get removed? I suggested this to @costin-eseanu because I was worried about the possibility of a typo, but maybe it's something we can no-op by default?

Reply:
I don't think the build times would be visibly affected by the 20 or so extra string comparisons at compile time. :)